yarpl/include/yarpl/flowable/Flowable.h

```diff
@@ -95,6 +95,8 @@ class Flowable : public virtual Refcounted {
 
   auto take(int64_t);
 
+  auto skip(int64_t);
```

Comment:
just copied the signature for take. `int64_t` is being used in a few other places, and we're using these numbers with `request(int64_t)`, and I wanted to try to eliminate implicit casts